### PR TITLE
feat: add infrared protocol

### PR DESCRIPTION
**Pull request by 0xFrontRunner**
## Gauge List Contribution

Please make sure you've completed the following before submitting your pull request:

- [x] I have read the [contribution guidelines](README.md)

## Summary by CodeRabbit

- **New Features**
	- Introduced a new protocol entry for "Infrared Finance" in the gauge list, providing details such as description, ID, image, name, and URL.
	- Highlighted the focus on simplifying interactions with Proof of Liquidity through liquid staking products.

### CODE REVIEW

**Review by multibear-infra — APPROVED** [review]
LGTM! Merge it!

File: src/gauges/bartio/defaultGaugeList.json [hunk 1]
```diff
@@ -138,6 +138,13 @@
   ],
   "name": "Infrared Bartio Default Gauge List",
   "protocols": [
+    {
+      "description": "Infrared is focused on building infrastructure around the Proof of Liquidity (PoL) mechanism pioneered by Berachain",
```
Comment (mlenser) [inline src/gauges/bartio/defaultGaugeList.json:142]:
```suggestion
      "description": "Infrared simplifies interacting with Proof of Liquidity with our liquid staking products.",
```

File: src/gauges/bartio/defaultGaugeList.json [hunk 2]
```diff
@@ -138,6 +138,13 @@
...
+      "id": "infrared",
+      "image": "infrared.svg",
+      "name": "Infrared Finance",
+      "url": "https://infrared.finance/"
```
Comment (mlenser) [inline src/gauges/bartio/defaultGaugeList.json:146]:
```suggestion
      "url": "https://infrared.finance"
```

**Review by mlenser — CHANGES_REQUESTED** [review]
Minor stuff

**Review by mlenser — APPROVED** [review]

**Pull request merged**